notebooks/demo-train-with-faq.ipynb

```diff
@@ -0,0 +1,839 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    ## To do: better split of docs
</pre><p> </p><p>I believe this comment/TODO is a remmant of the simple chunk splitter, and probably already satisfied by the use of <code>MarkdownTextSplitter</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>